.gitignore

```diff
@@ -56,3 +56,9 @@ ipch/
 *.opendb
 *.db
 coverage/
+# Gradle Files
++.gradletasknamecache
++.gradle/
++build/
++bin/
++.settings/ 
```

Comment:
The output directory appears to be called `out`. You've got the `.class` files in the commit right now :)

Reply:
You can put `.gitignore` files at any level to apply their patterns only to files below that directory. I wonder if we should move these (and add `out/`) to a `gitignore` in `clients/java`?